src/tests/test_parse.py

```diff
@@ -78,13 +78,26 @@ def test_parse_shovel_operator():
     }
 
 
-def test_parse_return_shovel_operator():
-    assert parse("dest= << SELECT * FROM work", empty_config) == {
+@pytest.mark.parametrize(
+    "input_string",
+    [
+        "dest= << SELECT * FROM work",
+        "dest = << SELECT * FROM work",
+        "dest =<< SELECT * FROM work",
+        "dest =        << SELECT * FROM work",
+        "dest      =<< SELECT * FROM work",
+        "dest =          << SELECT * FROM work",
+        "dest=<< SELECT * FROM work",
+    ],
+)
+def test_parse_return_shovel_operator(input_string):
```

Comment:
@edublancas 
Do we allow these cases?

`dest= <<SELECT * FROM work`

`dest<< SELECT * FROM work`

Reply:
I think we can add another test for more complex queries such:

```
%sql --save save_result result<< SELECT * FROM work limit 2
```

This query should store `save_result` and `result`.

(it works)

Reply:
I assume we do not allow the first one, since we need at least one space after `<<`

Reply:
> I think we can add another test for more complex queries such:
> 
> ```
> %sql --save save_result result<< SELECT * FROM work limit 2
> ```
> 
> This query should store `save_result` and `result`.
> 
> (it works)

What's the meaning of this test, since it seems does not have the `=<<` operator used in this PR, only `<<` operator